docs/conf.py

```diff
@@ -70,9 +70,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.1.0'
+version = u'0.0.6'
```

Comment:
@LefterisJP suggested versioning it like 0.1.0. I don't remember the reasoning behind it, but I am fine with 0.0.6 too.

Reply:
No that is just the current version from Konrad's deploy test. I think it's a good idea to bump to 0.1.0 for the preview.

Reply:
yes 0.1.0 is the preview, not now